lib/puppet/face/module/list.rb

```diff
@@ -17,6 +17,10 @@
       summary _("Whether to show dependencies as a tree view")
     end
 
+    option '--strict-semver' do
+      summary _('Whether version ranges should exclude pre-release versions')
+    end
+
```

Comment:
Might be clearer to set the default, like we do for the `puppet generate type --force` flag:

```ruby
default_to { false }
```